react/dist/shared.js

```diff
@@ -20,19 +20,23 @@ exports.typography = {
         fontWeight: 600,
     },
     body1: {
-        lineHeight: '1.5em',
+        lineHeight: '1.5',
     },
     body2: {
-        lineHeight: '1.43em',
+        lineHeight: '1.43',
     },
     button: {
         fontWeight: 600,
-        lineHeight: '1.75em',
+        lineHeight: '1.75',
     },
     overline: {
         letterSpacing: '2px',
-        fontSize: '12px',
+        fontSize: '0.75rem',
         fontWeight: 600,
+        lineHeight: '2.67',
+    },
+    caption: {
+        lineHeight: '1.67',
```

Comment:
This react dist file should already be updated correctly...you may need to merge in the latest changes from dev branch into yours.

Reply:
Merging dev into this branch got it all sorted out.